packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -117,6 +118,18 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
     return returnPrice;
   }
 
+  getHistoricalPricePeriods() {
```

Comment:
this method simply exposes data that was previously internal to the price feed. it's useful for things like developer mining to get all historic data.